pkg/humio/client_test.go

```diff
@@ -92,6 +92,23 @@ func TestClient(t *testing.T) {
 		_, err := testClient.ListRepos()
 		require.Nil(t, err)
 	})
+
+	//remove this test. its not a good functional test
```

Comment:
What was the goal of this test and can we look at re-implementing if it's essential?

Reply:
This was a test that @bossinc commented out - I can revisit it.